src/decrypt.ts

```diff
@@ -16,8 +16,6 @@ export default async (payload: DecryptSettings): Promise<Buffer> => {
   // Password
   if (payload.password) {
     callArguments.push(`--password=${payload.password}`);
-  } else {
```

Comment:
According to qpdf the c++ library you can just not include the password param if you don't have one.  Including it as an empty string doesn't seem to work and causes the qpdf library to throw an error even in the command line.  So I believe it has to do with how the single vs double quotes get interpreted by the command line.

I just removed the else.